pkg/meta/redis.go

```diff
@@ -164,7 +165,38 @@ func newRedisMeta(url string, conf *Config) (Meta, error) {
 	}
 
 	m.checkServerConfig()
-	return m, nil
+	m.root = 1
```

Comment:
delete this line

Reply:
this one is needed, otherwise the lookupSubdir() will fail